assets/javascripts/discourse/initializers/chat-keyboard-shortcuts.js

```diff
@@ -41,6 +51,22 @@ export default {
       api.addKeyboardShortcut("alt+down", handleMoveDownShortcut, {
         global: true,
       });
+
+      api.addKeyboardShortcut(
+        "ctrl+b",
+        (event) => modifyComposerSelection(event, "bold"),
+        { global: true }
+      );
+      api.addKeyboardShortcut(
+        "ctrl+i",
+        (event) => modifyComposerSelection(event, "italic"),
+        { global: true }
+      );
+      api.addKeyboardShortcut(
+        "ctrl+e",
```

Comment:
Are these `ctrl+?` on all platforms? If so, can we make it `cmd+?` on apple platforms like we do in d-editor?

https://github.com/discourse/discourse/blob/a48231041b3f533f9d26be6e68ed1cadaea93495/app/assets/javascripts/discourse/app/components/d-editor.js#L197-L206

Reply:
Ah yes they need to be dynamic per platform, ty will fix